src/general_manager/api/graphql.py

```
@@ -156,18 +165,38 @@ def createGraphqlInterface(cls, generalManagerClass: GeneralManagerMeta) -> None
+            attr_name,
+            attr_value,
+        ) in generalManagerClass.Interface.getGraphQLProperties().items():
+            raw_hint = attr_value.graphql_type_hint
+            origin = get_origin(raw_hint)
+            type_args = [t for t in get_args(raw_hint) if t is not type(None)]
+
+            if origin in (list, tuple, set):
+                element = type_args[0] if type_args else Any
+                if isinstance(element, type) and issubclass(element, GeneralManager):  # type: ignore
+                    graphene_field = graphene.List(
+                        lambda: GraphQL.graphql_type_registry[element.__name__]
```

Ruff (0.13.2)

Function definition does not bind loop variable `element`

(B023)